config/routes.rb

```diff
@@ -2791,7 +2791,11 @@ def create_api_route(verb, url, action)
                          "/api(/:version)/#{collection_name}(/*c_suffix)",
                          action_for_collection(collection_name, verb))
       else
-        create_api_route(verb, "/api(/:version)/#{collection_name}(/:c_id)", action_for(verb))
+        if collection.options.include?(:subcollection) || collection.subcollections
```

Comment:
@abellotti this condition will go away soon - it's just here to divide and conquer these collections